content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

```diff
@@ -7,6 +7,12 @@ slug: sig-usability-spotlight-2021
 
 **Author:** Kunal Kushwaha, Civo
 
+_**Note: SIG Usability is Not active.**_
+_SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
+As a result, the links and information provided in this post may no longer be valid or relevant.
+Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the k8s/community policy.
+We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._
```

Comment:
```suggestion
We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._
```
Instead of pointing out directly, maybe we can provide a reference link.

Reply:
what reference link? aren't they already on the kubernetes website if reading the blog🤔
or https://kubernetes.io/community/

Reply:
In other words we can refer them to _k/community_ repo by adding a reference ink at the last of the text you've writtten.